src/QEQ/fix_qeq.cpp

```diff
@@ -53,12 +55,12 @@ namespace {
 
 FixQEq::FixQEq(LAMMPS *lmp, int narg, char **arg) :
   Fix(lmp, narg, arg), list(nullptr), chi(nullptr), eta(nullptr),
-  gamma(nullptr), zeta(nullptr), zcore(nullptr), chizj(nullptr), shld(nullptr),
+  gamma(nullptr), zeta(nullptr), zcore(nullptr), qmin(nullptr), qmax(nullptr), omega(nullptr), chizj(nullptr), shld(nullptr),
```

Comment:
These variables are not used by the parent `FixQEq` class and therefore should be moved to the child `FixQEqCTIP` class instead.

Reply:
Actually, it is better to leave them in the base class or else the entire file parser would have to be duplicated